core/mrc.js

```diff
@@ -430,6 +432,12 @@ exports.getModule = class mrcModule extends MenuModule {
                 this.processOutgoingMessage(newmsg, cmd[1]);
                 break;
 
+            case 'me':
+                const memsg = cmd.slice(1).join(' ');
+                const handle = this.state.alias;
+                this.sendMessageToMultiplexer("", "", this.state.room, "|15* |13"+handle+" "+stripMciColorCodes(memsg));
```

Comment:
You can use format strings, e.g. 
```
`|15* |13{handle} {stripMciColorCodes(memsg))`
```

Reply:
interesting, I must have done something wrong as I tried that and the output just showed ".... {handle} {stripMciColorCodes(memsg)}" lol

Will have a play

Reply:
ok I figured out how to use those, dunno if that's how you meant but it works still =)